torch/csrc/jit/mobile/register_ops_common_utils.cpp

```diff
@@ -0,0 +1,122 @@
+#include <ATen/core/dynamic_type.h>
+#include <ATen/core/type_factory.h>
+#include <torch/csrc/jit/mobile/register_ops_common_utils.h>
+
+namespace torch {
+namespace jit {
+
+int64_t normalizeIndex(int64_t idx, int64_t list_size) {
+  if (idx < 0) {
+    // Handle negative indexing
+    idx = list_size + idx;
+  }
+  return idx;
+}
+
+C10_UNUSED at::Tensor to_dispatch(
```

Comment:
Any reason to move this function in a common file? I couldn't find the new usage.